style_traits/lib.rs

```diff
@@ -5,8 +5,6 @@
 //! This module contains shared types and messages for use by devtools/script.
 //! The traits are here instead of in script so that the devtools crate can be
 //! modified independently of the rest of Servo.
-
-#![crate_name = "style_traits"]
```

Comment:
https://doc.rust-lang.org/rust-by-example/attribute/crate.html

> it is important to note that both the crate_type and crate_name attributes have no effect whatsoever when using Cargo

So was this added in case that cargo is not used??

I'm a bit worried about accidentally breaking something that I'm missing.

Reply:
> So was this added in case that cargo is not used??

I think so. See: https://web.mit.edu/rust-lang_v1.25/arch/amd64_ubuntu1404/share/doc/rust/html/rust-by-example/attribute/crate.html Given the timing of this code (when Cargo was still only a few months old), I suspect it may have been written with a pre-cargo world in mind.

Reply:
This passes Mozilla CI (https://treeherder.mozilla.org/jobs?repo=try&revision=a0ba50e438a0bdedc8e36fa026fe9550d6e3dd71). Perhaps I should just submit this upstream?

Reply:
Yes submitting it upstream sounds better